game_patch/hud/multi_spectate.cpp

```diff
@@ -205,16 +205,15 @@ bool multi_spectate_execute_action(rf::ControlConfigAction action, bool was_pres
     return false;
 }
 
-void multi_spectate_on_player_death(rf::Player* player, rf::Player* killer)
+void multi_spectate_on_player_death(rf::Player* victim, rf::Player* killer)
```

Comment:
Should have `multi_spectate_on_player_kill` for consitency with `on_player_kill`.

Reply:
Good point. Done.